spec/manuals_publisher/removing_content_spec.rb

```diff
@@ -0,0 +1,59 @@
+feature "Removing content on Manuals Publisher", manuals_publisher: true do
+  include ManualsHelpers
+
+  let(:title) { title_with_timestamp }
+  let(:section_title) { title_with_timestamp }
+  let(:section_slug) { section_title.downcase.tr(" ", "-").gsub(/[^\w-]/, '') }
```

Comment:
might be a bit clearer if we just use the `slug_with_timestamp` method here rather than tweaking the title?

Reply:
The manuals publisher generates the slug from the title provided.

In this case, I think we could hardcode the title, as it doesn't need to be unique.